rebasehelper/application.py

```diff
@@ -23,20 +23,18 @@
 #          František Nečas <fifinecas@seznam.cz>
 
 import fnmatch
+import logging
 import os
 import shutil
-import logging
+from typing import Any, Dict, List, Optional, cast
```

Comment:
I see a lot of inconsistencies. There is no empty line here, but there is one in many other files.